src/table/newgrf_debug_data.h

```diff
@@ -680,7 +731,7 @@ static const NIFeature * const _nifeatures[] = {
 	&_nif_industry,     // GSF_INDUSTRIES
 	nullptr,               // GSF_CARGOES (has no "physical" objects)
 	nullptr,               // GSF_SOUNDFX (has no "physical" objects)
-	nullptr,               // GSF_AIRPORTS (feature not implemented)
+	&_nif_airport,      // GSF_AIRPORTS
```

Comment:
What kind of weird alignment did they pick here? You are consistent, I give you that, but this is weird :D (please leave it as it is :P It just triggered my OCD)

Reply:
It's a side effect of the bulk renaming NULL to nullptr

Reply:
Ha! Tnx for that, now my OCD is much less triggered :D